### PR TITLE
[CEDS-1130] - ip address whitelisting

### PR DESCRIPTION
**Pull request by raghu1978**

### CODE REVIEW

File: app/config/AppConfig.scala [hunk 1]
```diff
@@ -120,4 +123,28 @@ class AppConfig @Inject()(override val runModeConfiguration: Configuration, val
   private def feature2Key(feature: Feature): String = s"microservice.services.features.$feature"
 
   private def str2FeatureStatus(str: String): FeatureStatus = FeatureStatus.withName(str)
+
+  private def whitelistConfig(key: String): Seq[String] =
+    Some(
```
Comment (PatrykRudnicki) [inline app/config/AppConfig.scala]: Just wondering why we have `Some` here?

File: app/config/filters/ExcludingCSRFFilter.scala [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+class ExcludingCSRFFilter @Inject()(filter: CSRFFilter) extends EssentialFilter {
+
+  override def apply(nextFilter: EssentialAction): EssentialAction = new EssentialAction {
+
+    override def apply(rh: RequestHeader): Accumulator[ByteString, Result] = {
+      val chainedFilter = filter.apply(nextFilter)
+      if (rh.tags.getOrElse("ROUTE_COMMENTS", "").contains("NOCSRF")) {
+        nextFilter(rh)
+      } else {
+        chainedFilter(rh)
```
Comment (PatrykRudnicki) [inline app/config/filters/ExcludingCSRFFilter.scala]: Here we can just use: `filter.apply(nextFilter)`

We don't need to evaluate it before if, just only when we need it

File: conf/application.conf [hunk 3]
```diff
@@ -32,7 +32,9 @@ play.filters.csrf.contentType.whiteList = ["application/xml", "application/json"
 
 play.http.requestHandler = "uk.gov.hmrc.play.bootstrap.http.RequestHandler"
 play.http.errorHandler = "handlers.ErrorHandler"
-play.http.filters = "uk.gov.hmrc.play.bootstrap.filters.FrontendFilters"
+#play.http.filters = "uk.gov.hmrc.play.bootstrap.filters.FrontendFilters"
```
Comment (PatrykRudnicki) [inline conf/application.conf]: Can you remove this comment?

And base on this change I think that we can get rid of `FrontendFilters` too

File: app/config/AppConfig.scala (hunk 1 above)
Comment (raghu1978) [reply app/config/AppConfig.scala]: either Some or using Seq with similar situation with splitting the string

**Pull request merged**